src/components/App.vue

```diff
@@ -10,6 +10,7 @@ import AppNotification from '@/components/textElements/AppNotification.vue';
 import { initHotkeys } from '@/settings/hotkeysSettings';
 import checkAppVersion from '@/settings/appVersion';
 import { handleQueryOnLoad } from '@/router/utils';
+import { checkAndSetIsMobileScreen, debouncedCheckAndSetIsMobileScreen } from '@/store/utils';
```

Comment:
hmmmm, you use store utils in component?
I think it have to be different util
And also think about utils structure at all. I think it start to be messy there.

Reply:
store utils it's utils which can be use in store
but here is an action in store
we can add some module like Appearance and do it there